doc/doxygen-root/README.md

```diff
@@ -0,0 +1,60 @@
+# CBMC documentation
+
+This is the root of CBMC documentation.
+
+## Install doxygen
+
+Building this documentation requires `doxygen` and `graphviz`.
+
+* On MacOS,
+  we recommend installing with [brew](https://brew.sh/):
+  ```
+  brew install doxygen graphviz
+  ```
+* On Ubuntu, we recommend installing with using `apt`:
+  ```
+  sudo apt install doxygen graphviz
+  ```
+
+Note: As of
+
+## Build the documentation
+
+To build the documentation, run
+
+```
+doxygen
```

Comment:
There's also a `Makefile`.

I tried running the `default` action of the `Makefile` first, but got a very broken version of the documentation (probably because I was missing stuff from the dependencies?) - which one is the defacto way a developer builds documentation locally? Just running `Doxygen`, or the `Makefile`?

Reply:
Good catch.  There is now a makefile to automate the five or six invocations of doxygen.  The current build command is "make".